sbteclipse-plugin/src/sbt-test/sbteclipse/02-contents/project/Build.scala

```diff
@@ -106,7 +106,7 @@ object Build extends Build {
     "scala",
     new File("scala"),
     settings = Project.defaultSettings ++ Seq(
-      EclipseKeys.projectFlavor := EclipseProjectFlavor.Scala
+      EclipseKeys.projectFlavor := EclipseProjectFlavor.ScalaIDE
```

Comment:
Just to be clear, this PR doesn't change the behavior here. Also, I believe almost no user will have this line in their build because it's the default value.

I imagine that most users with Scala projects end up adding ScalaIDE support. I think it's better to be more explicit that what we're doing is adding ScalaIDE support though because it lets users know that if they set this option, then they also need to install ScalaIDE. It also makes it obvious that this is only changing settings related to ScalaIDE and not other settings, which isn't very obvious now. Finally, it means if there's ever a second Scala plugin (this happens. e.g. there are two popular Maven plugins) then there won't be confusion as to what setting Scala here means.